README.md

```diff
@@ -2,20 +2,30 @@
 
 Starter project for a chess-based TDD exercise.
 
-## Running the application
+## Getting the application up and running
 
-To run the application, first ensure that you have installed Poetry on your system. Install
-dependencies using the command ``poetry install``.
+### Installing dependencies
+First, let's ensure that you have installed python and pip on your system. 
 
-To run the application, use the command ``poetry run flask run`` and navigate to http://localhost:5000. You can reset the board at any time by restarting the server.
+- Run `python --version` and you should see a version number. If not, you may need to install Python on your machine. On ACG, this should be already available - ensure you're using the image specified in the instructions.
+- Run `pip --version` and you should also see a version number. If not, you may need to install pip on your machine. On ACG, you can do `sudo apt install python3-pip` to install pip.
```

Comment:
Another option is to try `python -m pip` like you've done with `pytest` below.

Reply:
`/usr/bin/python: No module named pip` :<